x-pack/plugins/apm/typings/numeral.d.ts

```diff
@@ -1,20 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
```

Comment:
@elastic/apm-ui This type definition was moved into `@elastic/numeral` so it could be easily shared with everyone automatically.